layer/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .flavors.custom import CustomModel  # noqa
 from .global_context import current_project_name  # noqa
 from .logged_data.callbacks import KerasCallback, XGBoostCallback  # noqa
+from .logged_data.log_data_runner import Image  # noqa
```

Comment:
Make sure to update this

Reply:
Good catch. I wonder why my unit tests didn't cover that. Fixed.